runatlantis.io/docs/server-configuration.md

```diff
@@ -229,10 +229,13 @@ Values are chosen in this order:
   atlantis server --data-dir="path/to/data/dir"
   ```
   Directory where Atlantis will store its data. Will be created if it doesn't exist.
-  Defaults to `~/.atlantis`. Atlantis will store its database, checked out repos, Terraform plans and downloaded
+  Defaults to `/home/atlantis/`. Atlantis will store its database, checked out repos, Terraform plans and downloaded
   Terraform binaries here. If Atlantis loses this directory, [locks](locking.html)
   will be lost and unapplied plans will be lost.
 
+  Note that the atlantis user is restricted to `/home/atlantis/`. 
```

Comment:
I'd mention the gosu portion in the atlantis-base image

Reply:
Is it not rather confusing to document this under the `--data-dir` flag?